src/codeGenerators/circuit/zokrates/toCircuit.ts

```diff
@@ -113,8 +113,11 @@ function codeGenerator(node: any) {
     case 'IndexAccess':
       return `${codeGenerator(node.baseExpression)}_${codeGenerator(node.indexExpression)}`;
 
+    case 'TupleExpression':
+      return `(${node.components.map(codeGenerator).join(` `)})`;
+
     case 'MsgSender':
-      return 'msgSender';
+      return 'msg';
```

Comment:
This is for the msg.sender address fix above to remove duplicates - now its always referred to in the output circuit as `msg` (before it was sometimes `msgSender`, sometimes `msg`)